src/features/classification/security-controls/utils.js

```diff
@@ -74,7 +77,19 @@ const getWatermarkingMessages = (controls: Controls): Array<MessageItem> => {
     const items = [];
     const isWatermarkEnabled = getProp(controls, `${WATERMARK}.enabled`, false);
     if (isWatermarkEnabled) {
-        items.push({ message: messages.watermarkingApplied });
```

Comment:
- initially I thought about just passing in `formattedCompMessage` and making `formattedCompMessage` and `message` optional -- but later noticed that the components that render `SecurityControlsItem` uses the `message.id` for id attribute. 
- The code became not too clean with that approach ^ so right now I defaulted to have a `message` prop that is the same copy but no link, so it can grab the id
- Within `SecurityControlsItem`, we will render `FormattedCompMessage` if it exists but fallback to `message` object. I can also add a boolean prop to `MessageItem` `useFormattedCompMessage` so in additional for just checking for existence of this prop, it'd be more explicit as well. 

^ Would want to hear others' input on what they think of this approach or if there is another preferred method as well. Thank you!